src/main/drivers/dshot.c

```diff
@@ -151,6 +151,19 @@ FAST_DATA_ZERO_INIT static float erpmToHz;
 FAST_DATA_ZERO_INIT static float dshotRpmAverage;
 FAST_DATA_ZERO_INIT static float dshotRpm[MAX_SUPPORTED_MOTORS];
 
+// Lookup table for extended telemetry type decoding (index = telemetryType >> 8)
+// Only contains extended telemetry types, eRPM is handled by conditional logic
+static const dshotTelemetryType_t extendedTelemetryLookup[8] = {
+    DSHOT_TELEMETRY_TYPE_TEMPERATURE, // 0x0200 >> 8 = 2
```

Comment:
It looks like  DSHOT_TELEMETRY_TYPE_x is actually naming `telemetryType` from `telemetryValue >> 9`. Using this table is possible, but simply documenting this property in `dshotTelemetryType_e` may be enough